tests/test_users.py

```diff
@@ -1,14 +1,13 @@
 import pytest
 import responses
 
-from pyartifactory import ArtfictoryUser
+from pyartifactory import ArtifactoryUser
 from pyartifactory.exception import UserAlreadyExistsException, UserNotFoundException
-from pyartifactory.models.Auth import AuthModel
-from pyartifactory.models.User import NewUser, UserResponse, SimpleUser
+from pyartifactory.models import AuthModel, NewUser, UserResponse, SimpleUser
 
 URL = "http://localhost:8080/artifactory"
 AUTH = ("user", "password_or_apiKey")
-SIMPLE_USER = SimpleUser(name="test_user")
+SIMPLE_USER = SimpleUser(name="test_user", uri="https:some.uri")

```

Comment:
That doesn't look like a URI ^^' (also could we change the SimpleUser model to use a `UrlStr` type? https://pydantic-docs.helpmanual.io/#exotic-types)